pkg/ros2/files/humble/vxscript

```diff
@@ -1,8 +1,9 @@
+xxx="/usr";iosViewPrune iosViewCurrent(), xxx, strlen(xxx)+1
 symlink "/ata4","/usr"
 shellConfigDefaultValueSet "VXE_PATH", "/usr/bin"
 rtpSpStackSize=0x100000
 taskSpawn 0,50,0,0x20000,setenv,"LD_LIBRARY_PATH","/usr/lib"
 taskSpawn 0,50,0,0x20000,setenv,"AMENT_PREFIX_PATH","/usr"
-taskSpawn 0,50,0,0x20000,setenv,"ROS_LOG_DIR","/ram"
+taskSpawn 0,50,0,0x20000,setenv,"ROS_LOG_DIR","/ram0"
 taskSpawn 0,50,0,0x20000,setenv,"RMW_IMPLEMENTATION","rmw_cyclonedds_cpp"
-cd "/usr/bin"
+cd "/usr/bin"
```

Comment:
I think you are missing EOL

Reply:
you might also remove this `if` statement to prevent the substitution
https://github.com/Wind-River/vxworks7-ros2-build/blob/humble-release-23.03/pkg/ros2/Makefile#L119

Reply:
and the README shall be updated accordingly. We do not need to mention 'ata0' anymore while starting QEMU

Reply:
> I think you are missing EOL

Not sure what do you mean by this?

Reply:
why there is a change here? Both lines are the same. Am I missing something?

```patch
- cd "/usr/bin"
+ cd "/usr/bin"
```

Reply:
Fixed.